src/commands/aksReconcileCluster/aksReconcileCluster.ts

```diff
@@ -0,0 +1,45 @@
+import * as vscode from 'vscode';
+import * as k8s from 'vscode-kubernetes-tools-api';
+import { IActionContext } from "@microsoft/vscode-azext-utils";
+import { reconcileUsingUpdateInCluster, determineProvisioningState, getAksClusterTreeItem } from '../utils/clusters';
+import { failed, succeeded } from '../utils/errorable';
+import { longRunning } from '../utils/host';
+
+export default async function aksReconcileCluster(
+  _context: IActionContext,
+  target: any
+): Promise<void> {
+  const cloudExplorer = await k8s.extension.cloudExplorer.v1;
+
+  const cluster = getAksClusterTreeItem(target, cloudExplorer);
+  if (failed(cluster)) {
+    vscode.window.showErrorMessage(cluster.error);
+    return;
+  }
+
+  const clusterName = cluster.result.name;
+
+  const clusterProvisioingState = await determineProvisioningState(cluster.result, clusterName);
```

Comment:
Looks like this is unused and could be removed...unless we can figure out the circumstances under which reconciliation is appropriate, and this is relevant?